notNeededPackages.json

```diff
@@ -355,10 +355,64 @@
             "asOfVersion": "2.0.0"
         },
         {
+            "libraryName": "ecmarkup",
```

Comment:
You only meant to add one package here, right? Git must have gotten confused due to #15264.